code/datums/quirks/negative_quirks.dm

```diff
@@ -889,6 +889,29 @@
 			return
 		quirk_holder.add_mood_event("wrong_cigs", /datum/mood_event/wrong_brand)
 
+/datum/quirk/item_quirk/chronic_illness
+	name = "Chronic Illness"
+	desc = "You have a chronic illness that requires constant medication to keep under control."
+	icon = FA_ICON_DISEASE
+	value = -8
+	mob_trait = TRAIT_CHRONIC_ILLNESS
+	gain_text = span_danger("You feel a bit off today.")
+	lose_text = span_notice("You feel a bit better today.")
+	medical_record_text = "Patient has a chronic illness that requires constant medication to keep under control."
+	hardcore_value = 6
```

Comment:
Only six is pretty rough, since it can randomly decide your new objective is to just make money to barely not die. Paraplegic and blind are 15, for reference. I think something like 10 would be more fair

Reply:
Agreed